src/eventbrite-provider.ts

```diff
@@ -9,22 +11,36 @@ function EventbriteProvider(this: any, options: any) {
 
   let API_KEY = ''
 
+  let eventbriteSDK: Sdk
```

Comment:
Since we are building many providers, let's establish a general convention that the SDK module should be referenced as the simple name of the provider. In this case, just `eventbrite`. Compare: https://github.com/senecajs/seneca-github-provider/blob/main/src/github-provider.ts#L24

Reply:
makes all sense, thanks